### ISSUE TITLE
Show a warning for skipped gems?

### ISSUE THREAD

**Issue by andyw8** [Shopify/tapioca#1347]
Tapioca [skips particular gems](https://github.com/Shopify/tapioca/blob/559f365e825e9160896c14041fcef1fc6cbc67c2/lib/tapioca/gemfile.rb#L155-L165), which can be confusing. I ran into this when trying to add RBIs for `debug`.

```
% tapioca gem debug
Requiring all gems to prepare for compiling...  Done

Error: Cannot find gem 'debug'
main % 
```

Perhaps we can output a warning when skipping a particular gem.

**Comment by drewhoffer** [comment]
I'd like to take a look at this if it's still available? Are we just wanting to show an output showing the [IGNORED_GEMS](https://github.com/Shopify/tapioca/blob/32163d328f3edf83883f6f84e9094d9ab141809c/lib/tapioca/gemfile.rb#L122) as output regardless on `tapioca gem`? Or should we show when someone is trying to run `tapioca gem debug` for example.

e.g.

![Image](https://github.com/user-attachments/assets/6b2eb89f-9457-4295-8f5c-1ad78e9e06d6)

**Comment by KaanOzkan** [comment]
> Or should we show when someone is trying to run tapioca gem debug for example

Yes this sounds good and feel free to take it on. I don't think we need to show it unless you specify the ignored gem name as an argument, so nothing on `tapioca gem`.

**Comment by written-fresh** [comment]
Hey, coming back to this after an hour... year... however long away. I'll dust this branch off and see if its still good to open.
